docs/sbft-formal-model/replica.i.dfy

```diff
@@ -360,10 +434,9 @@ module Replica {
     && v.workingWindow.prePreparesRcvd[seqID].Some?
     && var msg := v.workingWindow.prePreparesRcvd[seqID].value;
     // TODO: We should be able to commit empty (Noop) operations as well
-    && v' == v.(workingWindow := 
-               v.workingWindow.(committedClientOperations :=
-                                 v.workingWindow.committedClientOperations[msg.payload.seqID := 
-                                                                          Some(msg.payload.operationWrapper)]))
+    && v' == v.(committedClientOperations :=
+                                 v.committedClientOperations[msg.payload.seqID := 
+                                                                          msg.payload.operationWrapper])
   }
 
   predicate ContiguousCommits(c:Constants, v:Variables, targetSeqID:SequenceID)
```

Comment:
For better readability I'd call this `ContiguousCommitsUntil`. WDYT?

Reply:
Yes, we need to to be more concise, thanks.